src/cli/build.ts

```diff
@@ -68,64 +68,54 @@ export default function (program: RootCmd): void {
   program
     .command("build")
     .description("Build a Pepr Module for deployment")
-    .option("-e, --entry-point [file]", "Specify the entry point file to build with.", peprTS)
-    .option(
-      "-n, --no-embed",
-      "Disables embedding of deployment files into output module. Useful when creating library modules intended solely for reuse/distribution via NPM.",
+    .addOption(
+      new Option("-M, --rbac-mode <admin|scoped>", "Set RBAC mode.")
+        .choices(["admin", "scoped"])
+        .default("admin"),
     )
     .addOption(
       new Option(
-        "-i, --custom-image <custom-image>",
-        "Specify a custom image (including version) for Admission and Watch Deployments. Example: 'docker.io/username/custom-pepr-controller:v1.0.0'",
-      ).conflicts(["registryInfo", "registry"]),
+        "-I, --registry-info <registry/username>",
+        "Provide the image registry and username for building and pushing a custom WASM container. Requires authentication. Conflicts with --custom-image and --registry. Builds and pushes `'<registry/username>/custom-pepr-controller:<current-version>'`.",
+      ).conflicts(["customImage", "registry"]),
     )
+    .option("-P, --with-pull-secret <name>", "Use image pull secret for controller Deployment.", "")
     .addOption(
       new Option(
-        "-r, --registry-info [<registry>/<username>]",
-        "Provide the image registry and username for building and pushing a custom WASM container. Requires authentication. Builds and pushes `'registry/username/custom-pepr-controller:<current-version>'`.",
-      ).conflicts(["customImage", "registry"]),
+        "-c, --custom-name <name>",
+        "Set name for zarf component and service monitors in helm charts.",
```

Comment:
We have overlooked this in the first place but this also renames the helm chart so that you can deploy 2 separate Pepr modules.